fix(apple silicon): fixed various typos and product names (#6694)
scaleway/docs-content: pages/apple-silicon/how-to/remove-public-ip.mdx

After enabling Virtual Private Cloud (VPC) on your Mac mini, you can enhance the security of your server by further restricting external access.

> This guide walks you through a series of steps to ensure that your Mac mini is isolated from unauthorized access while maintaining necessary connectivity for management and maintenance.

Reviewer: ```suggestion
This guide shows you how to isolate your Mac mini from unauthorized access while keeping the connectivity needed for management and maintenance.
```
